lib/blacklight/configuration.rb

```diff
@@ -212,7 +212,14 @@ def repository_class
     end
 
     def search_builder_class
-      super || Blacklight::Solr::SearchBuilder
+      super || locate_search_builder_class
+    end
+
+    def locate_search_builder_class
+      ::SearchBuilder
+    rescue NameError
+      Deprecation.warn(Configuration, "Your application is missing the SearchBuilder. Have you run `rails generate blacklight:search_builder`? Falling back to Blacklight::Solr::SearchBuilder")
```

Comment:
Do we really need to deprecation `Blacklight::Solr::SearchBuilder`? It seems like it works fine as-is for common uses..